### DISCUSSION TITLE
'nuther silly question

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#570] [Q&A]
So,

I'm a HUGE fan of Midnight Commander. Many moons ago I actually bought a copy of Norton Commander for use on my dos boxes. As it turns out the developer of NC is the dev of MC. How cool is that?

Anywho, I've got MC installed on every one of my Linux boxes.  Frankly it's a godsend on my headless RPi installs, and that one of the motivations for this question.

Here is a screen grab of MC in a Putty window from a RPi (on the left) and an EOS terminal window (on the right):

![image](https://github.com/user-attachments/assets/ac25151b-e738-4f8d-89f2-7698e362d721)

The colors on in the Putty window are what I'm used to seeing.  Why are the colors in the EOS terminal so different?  They both are using the same display scheme.

How do I configure the EOS terminal window to display the "correct" colors?

Thanks!

**Comment by jeremypw** [comment]
I suspect you need to specify a custom value for (at least) the color "blue" in the elementary terminal.  Maybe MC only specifies the color name ("blue") rather than the precise color.  You can set up a custom color scheme from the Terminal AppMenu button in the headerbar.

**Comment by doobes** [comment]
Thank you.

That's much better
